CMakeLists.txt

```diff
@@ -1,5 +1,19 @@
 cmake_minimum_required(VERSION 3.1)
 
+# ouptut paths
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}")
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}")
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}")
+
+# Build shared libs
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+if(MSVC)
+    set(CMAKE_DEBUG_POSTFIX "d")
+endif()
+
+option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" ON)
```

Comment:
This is not effective if you do not remove `SHARED` from https://github.com/robotology/unicycle-footstep-planner/pull/13/files#diff-af3b638bc2a3e6c650974192a53c7291L49 .

Reply:
Sorry I just forgot it :grin: